rules/use-prefixes-to-improve-code-review-communication/rule.md

```diff
@@ -15,12 +15,12 @@ In the world of code reviews, ambiguity can lead to confusion, misunderstandings
 
 <!--endintro-->
 
-When conducting code reviews in a collaborative environment, it is essential to maintain effective communication. Utilizing prefixes in your comments, as suggested at [https://conventionalcomments.org](https://conventionalcomments.org/), can significantly enhance the code review process. Prefixes help convey the intent and impact of a comment, making it easier for the author and other team members to understand how to address it.
+When conducting code reviews in a collaborative environment, it is essential to maintain effective communication. Utilizing prefixes in comments, as suggested on [conventionalcomments.org](https://conventionalcomments.org), can significantly enhance the code review process. Prefixes help convey the intent and impact of a comment, making it easier for the author and other team members to understand how to address it.
 
-Let's have a look at the following example from Bob Northwind:
+Let's have a look at the following examples:
 
 :::greybox
-@bob-northwind
+Hi @bob-northwind
```

Comment:
```suggestion
@bob-northwind
```
This is an example Code Review comment from Bob Northwind, hence the clarification in the previous comment.